### DISCUSSION TITLE
Thumbs up (👍) can count as upvote

### DISCUSSION THREAD

**Discussion by fonsp** [community/community#3474] [Discussions]
Right now, there are two ways to "+1" a Discussion: you can react to the discussion message with 👍, or you can upvote (⬆️) the discussion message. 

**It would be great if these two actions were equivalent.** For example, reacting with 👍 could be counted as an upvote.

**Comment by PranamBhat** [comment]
It works the same way.

**Reply by fonsp** [reply]
This discussion is actually a good example of what I mean:
![image](https://user-images.githubusercontent.com/6933510/109960089-9d679400-7ce8-11eb-8a07-54098cd7a328.png)

**Reply by PranamBhat** [reply]
For the main discussion we can see this behavior already :

![image](https://user-images.githubusercontent.com/32120033/109972595-82634700-7d1d-11eb-9b79-a31a3da86487.png)


From the console, I can see that this behavior is existing somehow for the comments/reply section as well:


![image](https://user-images.githubusercontent.com/32120033/109972945-d53cfe80-7d1d-11eb-9508-47d257a7416b.png)

**Comment by PranamBhat** [comment]
There's some bug here @evi-liu @becca :


![image](https://user-images.githubusercontent.com/32120033/109973440-6613da00-7d1e-11eb-8fe9-7ba998b2a3d9.png)

**Comment by sebastianst** [comment]
I agree, or even just remove the emoji reactions. It is rather misleading that a :+1: is _not_ an upvote. The :arrow_up: hyperlink is easy to miss.

**Reply by 2br-2b** [reply]
Yes, I agree. Having both the :+1: and upvote buttons is confusing, so maybe just remove the :+1: and :-1: buttons, and leave the other emoji? That way, it encourages users to upvote/downvote so that the total votes are counted rather than being split.

I'd also be confused if I upvote a post one day, check it out later, and then see that I've reacted to it.

**Comment by PranamBhat** [comment]
Seems like this is a bug. @evi-liu @becca kindly have a look on prior basis :)

**Comment by apokusin** [comment]
Adding to the confusion, a "reply" to a comment seems to only allow emoji emotes. 

# Screenshot below 👇
<img width="876" alt="image" src="https://user-images.githubusercontent.com/1258230/157994589-0cfe86c2-8cf3-4789-9d7a-8d4bb22d47d1.png">

**Comment by Lee-Carre** [comment]
I, for one, value the distinction.

* 👍 signals approval
* ⬆️ signals importance or deserving of attention
* 👎 signals disapproval

Being able to do one, without the other, is useful.

I can imagine upvoting and giving a 👎, too.

**Reply by apokusin** [reply]
Consistency between posts and comments would be the only change required for either direction. You can see how people are using the thumbs up as an upvote for comments
